README.md

```diff
@@ -232,29 +197,15 @@ For examples:
   (3 rows)
   </pre>
 
-### Others
-- InfluxDB FDW supports pushed down some aggregate functions: count, stddev, sum, max, min.
-- InfluxDB FDW supports INSERT, DELETE statements.
-  - `time` and `time_text` column can used for INSERT, DELETE statements.
-  - `time` column can express timestamp with precision down to microseconds.
-  - `time_text` column can express timestamp with precision down to nanoseconds.
-- InfluxDB FDW supports bulk INSERT by using batch_size option from PostgreSQL version 14 or later.
-- WHERE clauses including timestamp, interval and `now()` functions are pushed down.
-- LIMIT...OFFSET clauses are pushed down when there is LIMIT clause only or both LIMIT and OFFSET.<br>
+## Pushdowning
 
-## Limitations
-- UPDATE is not supported.
-- WITH CHECK OPTION constraints is not supported.
-Following limitations originate from data model and query language of InfluxDB.
-- Result sets have different number of rows depending on specified target list.
-For example, `SELECT field1 FROM t1` and `SELECT field2 FROM t1` returns different number of rows if
-the number of points with field1 and field2 are different in InfluxDB database.
-- Timestamp precision may be lost because timestamp resolution of PostgreSQL is microseconds while that of InfluxDB is nanoseconds.
-- Conditions like `WHERE time + interval '1 day' < now()` do not work. Please use `WHERE time < now() - interval '1 day'`.
+- `WHERE` clauses including `timestamp`, `interval` and `now()` functions are pushed down.
+- InfluxDB FDW supports pushed down some aggregate functions: `count`, `stddev`, `sum`, `max`, `min`.
+- `LIMIT...OFFSET` clauses are pushed down when there is `LIMIT` clause only or both `LIMIT` and `OFFSET`.
```

Comment:
please add these new features

----

- Support pushdown DISTINCT argument for only count clause.
- Support pushdown ANY ARRAY.

Reply:
Added.

Reply:
Thank you.